persia/ctx.py

```diff
@@ -98,6 +209,7 @@ class TrainCtx(BaseCtx):
         world_size (int): world size of this cluster.
         num_forward_workers (int): worker num of sending forward request to servers.
         num_backward_workers (int): worker num of sending backward request to servers.
+        initial_embedding_dir(str): initial embedding dir, load checkpoint in this dir when enter TrainCtx.
```

Comment:
`emb_cpk_path` would be better.`initial_embedding_dir` sounds like a pretrained emb path.

Reply:
> `emb_cpk_path` would be better.`initial_embedding_dir` sounds like a pretrained emb path.

It is, a pre-trained embedding path, which will be loaded when first enter `TrainCtx`.